src/main/java/com/amihaiemil/versioneye/MkVersionEye.java

```diff
@@ -88,7 +106,29 @@ public VersionEye trusted() throws IOException {
 
     @Override
     public Me me() {
+        if(this.authenticated == null) {
+            throw new IllegalStateException("No Authenticated User.");
+        }
         return new MkMe(this.server, this.authenticated);
     }
+    
+    /**
+     * Add authenticated user to the MkServer.
+     */
+    private void addAuthenticatedUserToServer() {
+        JsonArray array = this.server.storage().build()
+            .getJsonArray("authenticated");
+        if(array == null) {
```

Comment:
@amihaiemil For this one I found that only ``getString()``, ``getInt()`` and getBoolean()`` have default values other don't :D